src/main/java/racingcar/Input.java

```diff
@@ -5,38 +5,44 @@
 import java.util.Arrays;
 
 public class Input {
+    private static final int MAX_NAME_LENGTH = 5;
+
     private Input() {
 
     }
 
-    public static ArrayList<String> inputName() {
+    public static ArrayList<String> readName() {
         String names = Console.readLine();
         return splitName(names);
     }
 
     public static ArrayList<String> splitName(String names) {
-        ArrayList<String> nameList = new ArrayList<>(Arrays.asList(names.split(",")));
-        checkName(nameList);
-        return nameList;
+        ArrayList<String> splitNames = new ArrayList<>(Arrays.asList(names.split(",")));
+        checkName(splitNames);
+        return splitNames;
     }
 
     public static void checkName(ArrayList<String> nameList) {
         for (String name : nameList) {
-            if (name.length() > 5) {
+            if (name.length() > MAX_NAME_LENGTH) {
                 throw new IllegalArgumentException("자동차 이름은 5글자까지 입력 가능합니다.");
             }
         }
     }
 
-    public static int inputNumber() {
+    public static int readNumber() {
         String number = Console.readLine();
         checkNumber(number);
         return Integer.parseInt(number);
     }
 
     public static void checkNumber(String number) {
-        if (!isNumber(number) || Integer.parseInt(number) <= 0) {
-            throw new IllegalArgumentException("1 이상의 올바른 수를 입력해주세요.");
+        if (!isNumber(number)) {
+            throw new IllegalArgumentException("숫자를 입력해 주세요.");
+        } else {
+            if (Integer.parseInt(number) <= 0) {
+                throw new IllegalArgumentException("1 이상의 수를 입력해 주세요.");
+            }
         }
```

Comment:
throw 를 하면 자동으로 아래 코드가 실행되지 않기 때문에 다음과 같이 리팩토링 해볼 수 있을 것 같아요 :)

```suggestion
        if (!isNumber(number)) {
            throw new IllegalArgumentException("숫자를 입력해 주세요.");
        } 
        if (Integer.parseInt(number) <= 0) {
            throw new IllegalArgumentException("1 이상의 수를 입력해 주세요.");
        }
```

Reply:
조언해 주신대로 리팩토링 해보았습니다!!

예외를 처리하는 부분이 익숙치 않다보니 해당 부분 진행할 때 앞으로 더욱 신경쓰겠습니다!